torch/csrc/jit/serialization/callstack_debug_info_serialization.cpp

```diff
@@ -151,6 +156,16 @@ c10::optional<ModuleInstanceInfo> InlinedCallStackDeserializer::
   // type_name might be empty string ""
   // In that case type_ptr should be just nullptr
   auto type_ptr = cu->get_class(type_name);
+  if (!type_ptr) {
+    // We may have lost type information. For example in lowered backends
+    // original class type has no relevance.
+    // However, to correlate ops to their original modules
+    // we saved both type name and instance name.
+    // In such cases, when module is absorbed by lowered backend
+    // we augment instance name with type name instead of losing it.
+    type_name = type_name.substr(type_name.find_last_of('.') + 1);
```

Comment:
Can `type_name.find_last_of('.')` be `string::npos` (i.e. not found)?

Reply:
Good catch. Yes it can be.